source/docs/appendices/migration-guides.md

```diff
@@ -7,6 +7,217 @@ This area will contain the Migration steps to follow for upgrading your store to
+* GraphQL transforms such as `FilterRootFields` must now be imported from `@graphql-tools/wrap` instead of `graphql-tools`.
+
+Here is an example from [Front-Commerce's Magento 2 GraphQL remote schema module's migration](https://gitlab.com/front-commerce/front-commerce/-/commit/964cb0dee8b614053c9147997f26c07327e61a0a#e9c8fef58455edd870533da439f028dca7618e8c_1_1):
+
+```diff
+- import { FilterRootFields } from "graphql-tools";
+- import authenticateRequest from "./authenticateRequest";
++ import { FilterRootFields } from "@graphql-tools/wrap";
++ import makeExecutor from "server/core/graphql/makeExecutor";
++ import withMagentoAuthorizationHeaders from "./withMagentoAuthorizationHeaders";
+import log from "../../../../scripts/log";
+
+// […]
+
+    const uri = config.magento.endpoint + "/graphql";
+
+    return {
+      uri: uri,
+      transforms: [
+        new FilterRootFields(
+          (operation, rootField) =>
+            operation === "Query" && rootField === "storeConfig"
+        ),
+      ],
+-      linkContextBuilders: [authenticateRequest()],
++      executor: makeExecutor(uri, {
++        fetchOptionsAdapter: withMagentoAuthorizationHeaders(),
++      }),
+    };
+  },
+};
+```
+
+### `axios`: double-check custom interceptors
+
+`axios` has been updated from 0.15.3 to 0.19.2.
+
+The most notable breaking change is related to how duplicate headers are handled (see https://github.com/axios/axios/pull/874 and [the whole changelog](https://github.com/axios/axios/blob/master/CHANGELOG.md) if interested). It should not impact your application though.
+
+Another low-level change from 0.17 is that the base url is now prepended to the `config.url` [**AFTER** interceptors (and not *BEFORE*)](https://github.com/axios/axios/pull/950/files#diff-91dcec0516f33811ee5fa71297160b3bL40). **If your app relies on custom interceptors, please ensure they are still working correctly.**
+You can use the `finalUrlFromConfig` helper function from a helper module added to mimic `axios` core feature. See [this commit](https://gitlab.com/front-commerce/front-commerce/-/commit/3fff1a398f7b33d56546b987e15c5f5dc6d43524#24dda67967f430db4a7fe8010764e27d5d7d01b6_63_67) for an example.
+
+### `react-intl`: changes for strings containing HTML tags
+
+`react-intl` was updated from 3.x to 4.x. The main breaking change is that missing tags will now throw errors if your i18n messages contain HTML tags. Please [pass additional rendering parameters for these tags](https://formatjs.io/docs/react-intl/upgrade-guide-4x#migrating-off-embedded-html-in-messages) in your `intl.formatMessage` calls.
+
+```diff
+- intl.formatMessage("Hello <strong>Bob</strong>");
++ intl.formatMessage("Hello <strong>Bob</strong>", {
++   strong: (text) => <strong>{text}</strong>,
```

Comment:
IIRC the only place where we've used this feature in the base theme is for the Cart Recap. It might be nice tell the users to look for them? I feel like this part does not explain what is expected from our users.

I'd say:
* they should check your translation files and look for html tags
*  they should get the ids of the relevant translations and check that the html tags is correctly referenced where these ids are used.
* we provide the list of the ids that by default have an html tag in front-commerce and tell the users to ensure they don't have any files using these ids or change these files if needed

(my sentences are pretty bad but that's the idea of it)